.github/workflows/node-ci.yml

```diff
@@ -7,14 +7,6 @@ on:
     # The rest are the defaults.
     types: [edited, opened, synchronize, reopened]
 
-# Our jobs run like this to minimize wasting resource cycles:
-#   1. Prime caches for primary configuration.  This way subsequent jobs
-#      can run in parallel but rely on this primed cache.
-#   2. Lint and coverage
-#     a. Lint
-#     b. Coverage
-#   3. Run tests for remaining configurations
-#      Since these don't share caches, we don't need to prime those caches.
 jobs:
   lint:
     name: Lint, flow, and coverage check
```

Comment:
Right now this is not a coverage check, right?  Just lint+flow+tests.

Reply:
That's right.  I'll change the name.